### ISSUE TITLE
Always lock on your teammates at close range

### ISSUE THREAD

**Issue by dclemon** [Franklin-Zhang0/Yolo-v8-Apex-Aim-assist#21]
Can you share a model that supports the identification of teammates? the outline of teammates will have white strokes, so theoretically, AI can fully support the distinction between enemies and teammates.

**Comment by Franklin-Zhang0** [comment]
No, this project is not for you to cheat. It's only for study and learn.
